src/allotropy/parsers/moldev_softmax_pro/softmax_pro_parser.py

```diff
@@ -88,33 +69,42 @@ def to_allotrope(self, named_file_contents: NamedFileContents) -> Model:
 
     def _get_model(self, file_name: str, data: Data) -> Model:
         return Model(
-            field_asm_manifest="http://purl.allotrope.org/manifests/plate-reader/BENCHLING/2023/09/plate-reader.manifest",
+            field_asm_manifest="http://purl.allotrope.org/manifests/plate-reader/REC/2024/06/plate-reader.manifest",
             plate_reader_aggregate_document=PlateReaderAggregateDocument(
                 device_system_document=DeviceSystemDocument(
                     device_identifier=NOT_APPLICABLE,
                     model_number=NOT_APPLICABLE,
                 ),
                 data_system_document=DataSystemDocument(
+                    ASM_file_identifier=NOT_APPLICABLE,
```

Comment:
ASM_file_identifier/unc_path/data_system_instance_identifier are not required.

Why are we filling them with N/A instead of leaving them out?